security/plaintext-ldaps-auth-control-center/confluent-platform-ccc-ldap.yaml

```diff
@@ -0,0 +1,131 @@
+---
+apiVersion: platform.confluent.io/v1beta1
+kind: Zookeeper
+metadata:
+  name: zookeeper
+  namespace: confluent
+spec:
+  replicas: 3
+  image:
+    application: confluentinc/cp-zookeeper:7.2.1
+    init: confluentinc/confluent-init-container:2.4.1
+  dataVolumeCapacity: 10Gi
+  logVolumeCapacity: 10Gi
+---
+apiVersion: platform.confluent.io/v1beta1
+kind: Kafka
+metadata:
+  name: kafka
+  namespace: confluent
+spec:
```

Comment:
might be cool to have ldap auth type on the listeners, to make this a full ldap example. i dont think we have an ldap auth example yet